service/history/workflow/mutable_state_impl.go

```diff
@@ -99,7 +99,7 @@ var (
 )
 
 type (
-	mutableStateBuilder struct {
+	MutableStateImpl struct {
```

Comment:
This name is definitely better than old one but I don't like `Impl` suffix. From the other side, I can't figure out the better name.